TelCo.ColorCoder/Part1.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+namespace TelCo.ColorCoder
+{
+   internal class ColorPair
+        {
+            internal Color majorColor;
+            internal Color minorColor;
+            public override string ToString()
+            {
+                string str = "";
```

Comment:
This is wrong place to put ToString.
This ToString is for the color pair class.

You need to introduce a new method for display all the color combinations. 

Another Design guideline: 
Also causes bidirectional referening.
ValueOfColorPair refers to ColorPair and 
ColorPair refers to ValueOfColorPair, not a good practice